microprofile/microprofile-rest-client-1.3/src/test/java/org/wildfly/swarm/ts/microprofile/rest/client/v13/MicroprofileRestClient13Test.java

```diff
@@ -0,0 +1,46 @@
+package org.wildfly.swarm.ts.microprofile.rest.client.v13;
+
+import java.io.IOException;
+
+import org.apache.http.client.fluent.Request;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.arquillian.DefaultDeployment;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Arquillian.class)
+@DefaultDeployment
+public class MicroprofileRestClient13Test {
+    /**
+     * microprofile-config.properties contains desired key "myClients":
+     * myClients/mp-rest/uri=http://localhost:8080
+     * that is used for @RegisterRestClient(configKey="myClients") ClientAutoCloseable
+     * that is called on url../config-key
+     */
+    @Test
+    @InSequence(1)
+    @RunAsClient
+    public void configKeyTest() throws IOException {
+        String response = Request.Get("http://localhost:8080/rest/client/config-key").execute().returnContent().asString();
+        assertThat(response).isEqualTo("Hello from endpoint");
+
+        // client with non-existing configKey
+        response = Request.Get("http://localhost:8080/rest/client/missing-config-key").execute().returnContent().asString();
+        assertThat(response).contains("Neither baseUri nor baseUrl was specified");
```

Comment:
Should we have two distinct tests? One for existing config key and one for nonexistent config key?